hknweb/events/views.py

```diff
@@ -105,3 +108,11 @@ def add_event(request):
             messages.success(request, 'Something went wrong oops')
             return render(request, 'events/add_event.html', {'form': EventForm(None)})
     return render(request, 'events/add_event.html', {'form': EventForm(None)})
+
+@method_decorator(permission_required('events.change_event', login_url='/accounts/login/'), name='dispatch')
+class EventUpdateView(generic.edit.UpdateView):
+    model = Event
+    fields = ['name', 'slug', 'start_time', 'end_time', 'location', 'event_type',
+              'description', 'rsvp_limit']
+    template_name_suffix = '_update'
```

Comment:
Why is the template named `event_update` if it's at `/events/<id>/edit`?
(Is some kind of consistency worth it?)

Reply:
> Approved. I'd like to see the message code (which seems frequently copy+pasted) moved into base.css and base.html, and perhaps the .content css as well, but otherwise pretty sound.

The message might appear in different places though for different pages

Reply:
I guess we could change it to have a default place to put messages? Seems too complicated for this pull request though

Reply:
> Why is the template named `event_update` if it's at `/events/<id>/edit`?
> (Is some kind of consistency worth it?)

I guess, the reason was that the django terminology is "update" but in hkn we tend to say "edit"

Reply:
It's already inconsistent lol, add_event versus event_edit